### DISCUSSION TITLE
Remove MC dousing

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#3358] [Suggestions]
Small suggestion:
Remove the dousing requirement and disable the quest from the Duke in Aszhara.

Reasons:
- the dousing isn't blizzlike and was just introduced for the vanilla feeling.
- people are still running MC, thus it can be actively tested if the auto-dousing works correctly.
- disabling the quest will also make the achievement unavailable as it is supposed to be in Wotlk. It is a direct reward (having the achievement) for those that did "participate" in the douse-testing and will not be available for new joiners (which would be blizzlike).
- convenience for raids, especially for new players or groups joining the realm.

**Comment by Annamaria-CC** [comment]
Hey Balleny,

Thanks for your suggestion. At this point of time we are not considering removing attunements. The planning for them to be removed will be when we reach the TBC phase. This would be more in line with a "blizzlike" progression.
